### ISSUE TITLE
[BUG] Third Monitor Lost Time.

### ISSUE THREAD

**Issue by ZeroPoke** [martinet101/ElevenClock#48]
When I installed this on Launch day of Windows 11, I have time on all three monitors.
Now I only have time on monitor 1 and 2.

**To Reproduce**
No idea.  I just noticed it when I got up this morning.  and it was still gone after a reboot.

**Expected behavior**
Time on all three monitors.

**Screenshots**
https://i.pl0x.in/1e3a4a034ab2948607e2c837/bfdff7be.png

**Comment by martinet101** [comment]
Check release 1.9 and install the version with the alignment fix.
Then follow the instructions in the last comment on issue #40

**Comment by martinet101** [comment]
Yes, this is a bug that affects only some users, i'm trying to figure out what is happening...

**Comment by urifoox** [comment]
I don't think that's true - my main monitor is 6 works fine on mine.

![image](https://user-images.githubusercontent.com/22828430/136816382-dfef4fd5-001e-48a0-b318-8c4c7b091370.png)

**Comment by martinet101** [comment]
Hey, i've fixed this!!! See https://github.com/martinet101/ElevenClock/issues/123.

Fix coming with 2.2
